documents/requirements/SRS.md

```diff
@@ -0,0 +1,117 @@
+### 1.1 Purpose
+This document specifies the requirements of our application. It also describes nonfunctional requirements, designconstraints, and other factors necessary to provide a complete and comprehensive description of the requirements for AMOGUS.
+### 1.2 Scope
+AMOGUS is a web application for people to practice their Mathematic skills similar to the well-known language learning app Duolingo. Each User will be able to have an account in order to keep track of their progress. Admins will be able to manage the Exercises as well as giving support to the users [see Use-Case-Diagramm].<br>
+Planned Systems are:
+- Gameplay System
+- Streak System
+- Level System
+- Accounts
+- Stats tracking
+
+### 1.3 Definition, Acronyms and Abbreviations
+| Abbreviation  | Explanation                               |
+|---------------|-------------------------------------------|
+| PAMO          | Project AMOGUS - The name of the Project  |
+| AMOGUS        | Advanced Mathematic Operations and Geometrics Under Stress |
+| UCS           | Use Cases                                 |
+
+### 1.4 References
+| Title                                                         | Date       | Publisher |
+|---------------------------------------------------------------|------------|-----------|
+| [AMOGUS Blog](https://github.com/CUMGroup/AMOGUS/discussions) | 19.10.2022 | CUMGroup  |
+| [Github](https://github.com/CUMGroup/AMOGUS/)                 | 19.10.2022 | CUMGroup  |
+
+### 1.5 Overview
+This document provides an overview over the project, its use cases and requirements
+
+## 2. Overall Description
+### 2.1 Vision
+AMOGUS is a web application for people to practice their Mathematic skills similar to the well-known language learning app Duolingo.
+### 2.2 Use Case Diagram
+![Use Case Diagram](UseCaseDiagramImage.png)
+### 2.3 Technology Stack
+The technology used will be:
+- Backend: ASP.Net Web API, Sqlite Database
+- Frontend: Angular Web app
+- IDE: Visual Studio 2022, Visual Studio Code
+- Project Management: Jira, Github, Discord
+
+## 3. Specific Requirements
+### 3.1 Functionality
+#### Accounts (Login, Registering, Profiles)
+We want to let the user create their own account to track their progress, play games and get notified if the would lose their streak.
+#### Game loop
+This is the most important feature of our app. Here we will implement the main Excercises in a playful way. Players get a random Math problem from a specific category and can then try to solve it to get points and progress in the game.<br>
+Currently planned categories are:
+- Mental Arithmetic
+- Analysis (with Differentiation and Integration)
+- Algebra (with Matrices)
+- Complex Number Arithmetic
+
+#### Streak System
+Here we want the player to get attached to the game more by introducing a streak system. The player gets a streak wich tracks the amount of consecutive days they (uninterupted) played. So if a player doesn't play for 24 hours, the streak gets reset.
```

Comment:
Change "So if a player doesn't play for 24 hours" to " So if a player doesn't play till midnight"

Reply:
fixed